### DISCUSSION TITLE
How tied to CKEditor5 is Trilium?

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#3972] [General]
It seems like Trilium relies heavily on CKEditor, but I'm not sure how tight this reliance is.
If CKeditor went away, would Trilium go with it, or would it be possible to use an alternative editor?

I'm also wondering since it seems like most new CKEditor features are being released as Premium features which have an added cost.
There are some no-cost alternatives like https://alex-d.github.io/Trumbowyg/ that have not paid/premium tier but still offer a decent feature set. Demos of many of the Trumbowyg features can be seen here: https://alex-d.github.io/Trumbowyg/demos/

**Comment by zadam** [comment]
Trilium could exist without CKEditor, but some features might be limited. There's quite a lot of them available, with CKEditor being the best open source one to my knowledge.

**Reply by meichthys** [reply]
Thanks!
